src/styles/booksStyles.css

```diff
@@ -0,0 +1,211 @@
+.book-container {
+  margin: 0 6rem 0 6rem;
+  display: flex;
```

Comment:
- [ ] You could consider using a reusable class for the `flex` property as it reoccurs many times in your codebase

Reply:
Ok, thanks for the suggestion! I created the `.flex` class ✔️